modules/tinymce/src/core/test/ts/browser/newline/InsertNewLineTest.ts

```diff
@@ -18,6 +19,21 @@ describe('browser.tinymce.core.newline.InsertNewLineTest', () => {
     InsertNewLine.insert(editor, args as EditorEvent<KeyboardEvent>);
   };
 
+  const setSelectionToBody = (editor: Editor, offset: number) => {
```

Comment:
This function is not really needed use `TinySelection.setCursor(editor, [], 1)` should do the same thing.

- [ ] Replace with mcagar function

Reply:
The setCursor function does not do the same thing, no, I started with it and used this version only because there was no other choice. It moves the cursor up and away from the body in this situation, invalidating the whole test.

Reply:
Then add a comment why it's not the same thing. The only difference I can think of is that Firefox renders a fake caret after that table.